scripts/check_version

```diff
@@ -6,26 +6,34 @@ TEMP_IOS_VAR=`perl -n -e '/MARKETING_VERSION = (.*);/ && print "$1\n"' ./ios/Hat
 IOS_PROJECT_VERSION_1=`echo $TEMP_IOS_VAR | awk '{print $1}'`
 IOS_PROJECT_VERSION_2=`echo $TEMP_IOS_VAR | awk '{print $2}'`
 
+# This will read everything of the string up to the first dash.
+IFS='-'
```

Comment:
Why not `STRIPPED_PACKAGE_VERSION=$(echo $PACKAGE_VERSION| cut -d- -f1)`?

Reply:
I took this from a split script I had, but i'll update to using `cut` since its more concise and clearer